openbb_sdk/sdk/core/pyproject.toml

```diff
@@ -10,8 +10,8 @@ packages = [{ include = "openbb_core" }]
 python = "^3.8"
 uvicorn = "^0.20.0"
 importlib_metadata = "^6.8.0"
+pandas = "~1.5.3"
```

Comment:
This should explicitly be >= in order to allow dep solving for both pandas 1.5 and 2

~ leads to openbb-core (0.0.0a7) depending on pandas (>=1.5.3,<1.6.0)

Reply:
true, thanks